person.rb

```diff
@@ -1,27 +1,16 @@
-require_relative 'nameable'
-
-class Person < Nameable
-  attr_accessor :name, :age
-  attr_reader :id
-
-  def initialize(name, age, parent_permission: true)
-    super()
+class Person
+  def initialize(age, name = 'Unknown', parent_permission = 'true')
     @id = Random.rand(1..1000)
     @name = name
     @age = age
     @parent_permission = parent_permission
   end
```

Comment:
- [ ] A method for adding rentals should be present. You can refer to this [lesson](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship) to learn more

Reply:
- [ ] @OpondoG as previously requested kindly add a method to add rentals in your person class.